auth/index.js

```diff
@@ -8,7 +8,7 @@ const { JWT_CERT } = process.env;
 
 export function verifyWebToken(ctx) {
   log('Verifying token');
-  const token = ctx && ctx.headers && ctx.headers.authorization;
+  const token = ctx && ctx.headers && ctx.headers.Authorization;
```

Comment:
This change will mean that `token` will always be `false` as `Authorization` is not available in `headers`

Reply:
Can be mitigated with a `ctx && ctx.headers && (ctx.headers.Authorization || ctx.headers.authorization)`.

But I would like to know how is this being set in the first place.